guides/saas/getting-started.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Getting started with Coder's hosted beta
+description: Get started with a hosted Coder deployment.
+---
+
+To help you get started, we've put together this guide to help you understand
+how the Coder hosted beta works and how to access your deployment.
+
+## What is Coder hosted beta?
+
+The Coder hosted beta is a hybrid cloud offering of Coder's self-hosted
+platform. With this model, Coder hosts the control plane, and you host your
+compute and development workspaces. The benefits include:
+
+- Automatic upgrades
+- No installation or DNS configuration
+- Managed logging & monitoring
+
+Curious how this works? Here's a breakdown of the architecture:
+
+**Coder's infrastructure**:
+
+- The `coderd` service - Responsible for rendering the dashboard UI,
+  provisioning workspaces and user authentication, including a variety of other functions
+  - Your `coderd` instance is _not_ shared across other Coder accounts. Each deployment
+      receives their own `coderd` instance
+- A PostgreSQL DB - Stores metadata related to your Coder instance,
+  such as user information, session tokens, etc.
+  - Coder uses a single PostgreSQL instance with each deployment having their
+    own database within the instance
+
+**Your infrastructure**:
+
+- Kubernetes cluster - Hosts your Coder workspaces, in addition to your source
+  code, which is stored on [Persistent Volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/)
+  that are mounted to each workspace pod
+
+`coderd` connects to your infrastructure through a service account created in
+your cluster, enabling you to create workspaces in your cluster.
+
+## Accessing your Coder hosted beta
+
+1. Navigate to your deployment URL and login with the provided credentials:
```

Comment:
These details are currently emailed to the admin. Perhaps we should mention that here in case they forget what their URL or password is?

Reply:
You're right - in the current state, it assumes the reader knows they've been emailed the credentials. I can clarify.